packages/net/tcp_connection.pony

```diff
@@ -879,8 +861,8 @@ actor TCPConnection
             end
           end
 
-          if sum >= _max_size then
-            // If we've read _max_size, yield and read again later.
+          if sum >= _read_buffer_size then
```

Comment:
we should consider freeing "yield after" from buffer size in which case #3186 should be updated.